src/AWS.Deploy.CLI/Commands/TypeHints/S3BucketNameCommand.cs

```diff
@@ -37,17 +37,15 @@ private async Task<List<S3Bucket>> GetData()
 
         public async Task<object> Execute(Recommendation recommendation, OptionSettingItem optionSetting)
         {
-            const string NO_VALUE = "*** Do not select bucket ***";
             var currentValue = recommendation.GetOptionSettingValue(optionSetting);
             var buckets = (await GetData()).Select(bucket => bucket.BucketName).ToList();
 
-            buckets.Add(NO_VALUE);
```

Comment:
Why did you remove the ability to not select a bucket?

Reply:
Added typehint data like we discussed